tests/org.jboss.tools.maven.ui.bot.test/src/org/jboss/tools/maven/ui/bot/test/AbstractMavenSWTBotTest.java

```diff
@@ -221,6 +221,7 @@ public void convertToMavenProject(String projectName, String defaultPackaging, b
 		new WaitUntil(new ProjectExists(projectName));
 		pexplorer.getProject(projectName).select();
 		new ContextMenuItem("Configure","Convert to Maven Project").select();
+		AbstractWait.sleep(TimePeriod.DEFAULT);
```

Comment:
You should use waiting condition instead of abstract wait, like new WaitUntil(new JobIsRunning()), or something similar. Is there a reason for AbstractWait?

Reply:
There is no need for this after all. Removed.